expected/01_general.out

```diff
@@ -351,3 +351,10 @@ SELECT 4, COUNT(*) = 0 FROM "PoWA".powa_statements_history;
         4 | t
 (1 row)
 
+-- Test toast_tuple_target
```

Comment:
The comment should be clearer about what it's testing, ie we're not expecting any relation with some specific metadata to not have that reloption, otherwise the next person who adds a new table and have this fail won't understand the problem.  Also it would be good to have something closer to e.g. https://github.com/powa-team/powa-archivist/blob/master/sql/01_general.sql#L64-L80 so that if the test fail you actually know why it's failing.

Reply:
Fixed